### ISSUE TITLE
sync jupyterlab theme with OS client settings

### ISSUE THREAD

**Issue by xgdgsc** [jupyterlab/jupyterlab-desktop#734]
Can it sync jupyterlab light/dark theme with OS? When using browser with dark reader extension. It can sync with OS.

**Comment by JasonWeill** [comment]
@mbektas noted that there's a "Sync JupyterLab theme" option in prefs: https://github.com/jupyterlab/jupyterlab-desktop/issues/543#issuecomment-1355973951

Which theme option do you have selected in the Settings window? Which OS and which JupyterLab Desktop version are you running?

**Comment by xgdgsc** [comment]
Yes I have that enabled and not working on Windows with auto dark mode installed.  I use latest release jupyterlab-desktop with a remote ssh forwarded session. Does the option only work with local sessions?

**Comment by mbektas** [comment]
@xgdgsc yes, the sync theme is designed to work for locally launched sessions or remote sessions that were launched with `--expose-app-in-browser`.

**Comment by xgdgsc** [comment]
Thanks for the tip for --expose-app-in-browser . Would it be better if adding that tip to the settings UI?

**Comment by xgdgsc** [comment]
It seems only work when reopen the session. Can it be made work with working inside the session , when system switches, do the switch? Just like other apps like vscode/edge can do.

**Comment by mbektas** [comment]
@xgdgsc `--expose-app-in-browser` is mainly intended for testing launches and used by desktop app as well. It is not suggested for general use cases.
Some setting changes do not really require restarts and can be applied in place, but we didn't get chance to optimize. Something to improve in future.
